lib/stack_master/commands/apply.rb

```diff
@@ -121,7 +121,7 @@ def stack_options
         {
           stack_name: stack_name,
           parameters: proposed_stack.aws_parameters,
-          capabilities: ['CAPABILITY_IAM'],
+          capabilities: ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],
```

Comment:
Yep, I don't think we should set either one automatically, but perhaps require a flag (or a setting in `stack_master.yml`) to do so.